db/migrate/20151130215538_create_movies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMovies < ActiveRecord::Migration
+  def change
+    create_table :movies do |t|
+      t.string :name
```

Comment:
It's absolutely not necessary since you include the model validations, but you can optionally made the name be required at the database level: `t.string :name, null: false`